src/components/ProseCode.vue

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
This file has been simplified significantly and the only notable diff is the new directory.

We have moved the mermaid diagram rendering elsewhere.